src/main/java/uk/nhs/hee/trainee/details/api/TraineeProfileResource.java

```diff
@@ -146,13 +146,13 @@ public ResponseEntity<String> getVerifiedTraineeIds(@NotNull @RequestParam Strin
   }
 
   /**
-   * Get the account details for a trainee TIS ID.
+   * Get the details for a trainee TIS ID.
    *
    * @param tisId The TIS ID to search by.
-   * @return The matching trainee user account details, or not found if not found.
+   * @return The matching trainee user details, or not found if not found.
    */
-  @GetMapping("/account-details/{tisId}")
-  public ResponseEntity<UserAccountDetails> getTraineeAccountDetails(@PathVariable String tisId) {
-    return ResponseEntity.of(service.getTraineeAccountDetailsByTisId(tisId));
+  @GetMapping("/trainee-details/{tisId}")
```

Comment:
This would be a breaking change 😁 - so the version would need to go to 2.0.0

Reply:
(I presume you've checked any downstream services that might use the old endpoint: it can be a bit tricky to coordinate a switch-over to happen exactly simultaneously so the new endpoint is available at the same time as the downstream service start using it. If there is a (short) period where those services might get a 404, we need to be sure that's ok)

Reply:
I had a search on our services. It doesn't seems this endpoint is in used in other downstream services. But yeah, this change of endpoint means the trainee-details and trainee-notifications need to be deployed at the same time.

Reply:
If the notification can't reach the endpoint, there would be error in the logs. But I think it would need a manual resend of the message as the original message has been resumed. I am leaning to keep the endpoint as the original one to avoid potential data discrepancy (at least splitting out from the functional changes, so it would be tested/rolled back without coupling.)

Reply:
Yeah, that's probably a good way to go. I don't know how much traffic hits the notification service: in a 5-minute window it might be nothing, tbh - but its probably good practice to have a period of 'parallel running' with both endpoints until the old one can be decommissioned safely.

Reply:
Ah sorry, I wrote ☝️ before I saw you had made some code changes. Are you thinking to just retain the old endpoint? I had assumed you were going to set up a new one that handled the same call, and then remove the old one once it wasn't being used. But if you think that the old endpoint is good enough for now I can definitely live with that! 😁